docs/content/docs/dev/table/materialized-table/statements.md

```diff
@@ -0,0 +1,344 @@
+    REFRESH_MODE = 'FULL'
+    AS 
+    SELECT 
+        p.ds,
+        p.product_id,
+        p.product_name,
+        AVG(s.sale_price) AS avg_sale_price,
+        SUM(s.quantity) AS total_quantity
+    FROM 
+        paimon_catalog.db1.product_table p
+    LEFT JOIN 
+        paimon_catalog.db1.sales_table s
+    ON 
+        p.product_id = s.product_id
+    WHERE 
+        p.category = 'electronics'
+    GROUP BY 
+        p.ds, p.product_id, p.product_name
+```
+
+## Limitations
+
+- Does not support explicitly specifying columns
+- Does not support modified query statements
+- Does not support using temporary tables, temporary views, or temporary functions in the select query
+
+# ALTER MATERIALIZED TABLE
+
+```
+ALTER MATERIALIZED TABLE [catalog_name.][db_name.]table_name SUSPEND | RESUME [WITH (key1=val1, key2=val2, ...)] | REFRESH [PARTITION partition_spec]
+```
+
+ALTER MATERIALIZED TABLE is used to manage materialized tables. This command allows users to suspend and resume refresh pipeline of materialized tables and manually trigger data refreshes.
+
+## SUSPEND
+
+```
+ALTER MATERIALIZED TABLE [catalog_name.][db_name.]table_name SUSPEND
+```
+
+SUSPEND is used to pause the background refresh pipeline of the materialized table.
+
+**Example:**
+
+```sql
+-- Specify SAVEPOINT path before pausing
+SET 'execution.checkpointing.savepoint-dir' = 'hdfs://savepoint_path';
+
+-- Suspend the specified materialized table
+ALTER MATERIALIZED TABLE my_materialized_table SUSPEND;
+```
+
+<span class="label label-danger">Note</span>
+- When suspending a table in CONTINUOUS mode, the job will be paused using STOP WITH SAVEPOINT by default. You need to set the SAVEPOINT save path using [parameters]({{< ref "docs/deployment/config" >}}#execution-checkpointing-savepoint-dir).
+
+## RESUME
+
+```
+ALTER MATERIALIZED TABLE [catalog_name.][db_name.]table_name RESUME [WITH (key1=val1, key2=val2, ...)]
+```
+
+RESUME is used to resume the refresh pipeline of the materialized table.
+
+**Example:**
+
+```sql
+-- Resume the specified materialized table
+ALTER MATERIALIZED TABLE my_materialized_table RESUME;
+
+-- Resume the specified materialized table and specify sink parallelism
+ALTER MATERIALIZED TABLE my_materialized_table RESUME WITH ('sink.parallelism'='10');
+```
+
+## REFRESH
+
+```
+ALTER MATERIALIZED TABLE [catalog_name.][db_name.]table_name REFRESH [PARTITION partition_spec]
+```
+
+REFRESH is used to proactively trigger the refresh of the materialized table.
+
+**Example:**
+
+```sql
+-- Refresh the entire table data
+ALTER MATERIALIZED TABLE my_materialized_table REFRESH;
+
+-- Refresh specified partition data
+ALTER MATERIALIZED TABLE my_materialized_table REFRESH PARTITION (ds='2024-06-28');
+```
+
+<span class="label label-danger">Note</span>
+- The REFRESH operation will start a Flink batch job to refresh the materialized table data. In CONTINUOUS mode, the materialized table needs to be paused first to avoid data inconsistency.
```

Comment:
I think we don't need to emphasize that `In CONTINUOUS mode, the materialized table needs to be paused first to avoid data inconsistency.`. The user can judge it.